### DISCUSSION TITLE
An exchange or public database of the material data

### DISCUSSION THREAD

**Discussion by PetersFozzy** [bow-simulation/virtualbow#259] [Ideas]
A possibility of exchange or a public database of the material data, divided into wood and fiberglass.

**Comment by stfnp** [comment]
A material database of some sort is already on the roadmap, see issue https://github.com/bow-simulation/virtualbow/issues/90. I like the idea of grouping the materials into wood and fiberglass (and maybe other categories), I've added this as a comment to that issue.

The materials will probably be stored in some kind of text file(s), so it should be possible to exchange them between users.
